src/main/java/it/smartcommunitylab/aac/config/SecurityConfig.java

```diff
@@ -21,6 +21,7 @@
 import it.smartcommunitylab.aac.core.entrypoint.RealmAwarePathUriBuilder;
 import it.smartcommunitylab.aac.core.provider.ProviderConfigRepository;
 import it.smartcommunitylab.aac.crypto.InternalPasswordEncoder;
+import it.smartcommunitylab.aac.openid.endpoint.EndSessionEndpoint;
```

Comment:
remove the unused import

Reply:
solved with 8136bdcd8efb357e56235b44e1fd6b3186c0582a